cosmic_ray/parsing.py

```diff
@@ -22,15 +23,21 @@ def get_ast(module):
         TypeError: If the source file for `module` can't be found.
     """
     try:
-        source = inspect.getsource(module)
-    except OSError:
-        LOG.warning("Unable to read source for module %s", module)
-        raise
-
-    try:
         source_file = inspect.getsourcefile(module)
     except TypeError:
+        source_file = ''
```

Comment:
This seems unnecessary since we're not going to do anything with `source_file` afterward, i.e. we're just going to re-raise the exception and exit the function.